cannot run build.ps1 without Visual Studio 2008 installed

The .Net Compact Framework parts can only be build with VS2008.

We should think about to drop support with next major release and add some conditionals to build scripts till end of support.

Nope, not enough:
```
antlr4cs\Runtime\CSharp\Antlr4.Runtime\Antlr4.Runtime.net35-cf.csproj(287,3): error MSB4019: Das importierte Projekt "C:\Windows\Microsoft.NET\Framework\v3.5\Microsoft.CompactFramework.CSharp.targets" wurde nicht gefunden. Vergewissern Sie sich, dass der Pfad in der <Import>-Deklaration korrekt und die Datei auf dem Datenträger vorhanden ist.
```

Sorry, but german system.

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -1,0 +1,30 @@
+# Building from Source
+
+## Prerequisites
+
+* Visual Studio 2015 (Community edition or higher)
+* Maven 3.3.3 or higher
+* Java 6 Runtime Environment (JRE)
+* Java 7 Development Kit
+* [Microsoft .NET Compact Framework 3.5](https://download.microsoft.com/download/c/b/e/cbe1c611-7f2f-4bcf-921d-2df718591e1e/NETCFSetupv35.msi)
+* [Power Toys for the Microsoft .NET Compact Framework 3.5](https://download.microsoft.com/download/f/a/c/fac1342d-044d-4d88-ae97-d278ef697064/NETCFv35PowerToys.msi)
+
+## Build environment
+
+| Environment Variable | Description | Example |
+| --- | --- | --- |
+| `M2_HOME` | Path to Maven | `J:\apps\apache-maven-3.3.3` |
+
+## Build script
+
+After the prerequisites are installed, the build is executed with the following commands:
+
+```powershell
+cd build
+.\build.ps1 -Verbosity minimal -GenerateTests
+cd ..
+```
+
+### Running tests
+
+Tests may be run using the commands seen in [appveyor.yml](appveyor.yml).

--- a/runtime/CSharp/Antlr4.sln
+++ b/runtime/CSharp/Antlr4.sln
@@ -8,6 +8,7 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{47C0086D-577C-43DA-ADC7-544F27656E45}"
 	ProjectSection(SolutionItems) = preProject
 		..\..\appveyor.yml = ..\..\appveyor.yml
+		..\..\CONTRIBUTING.md = ..\..\CONTRIBUTING.md
 		..\..\CreatingTargets.md = ..\..\CreatingTargets.md
 		..\..\tool\resources\org\antlr\v4\tool\templates\codegen\CSharp\CSharp.stg = ..\..\tool\resources\org\antlr\v4\tool\templates\codegen\CSharp\CSharp.stg
 		..\..\tool\src\org\antlr\v4\codegen\CSharpTarget.java = ..\..\tool\src\org\antlr\v4\codegen\CSharpTarget.java

